.github/workflows/smoke-checks.yml

```diff
@@ -287,3 +264,25 @@ jobs:
       if: always()
       run: bundle exec fastlane cocoapods_integration
       timeout-minutes: 25
+
+  automated-code-review:
+    name: Automated Code Review
+    runs-on: macos-12
```

Comment:
Shouldn't this be macOS-14?

Reply:
I thought it might make sense to run a code review on older Xcode so it can catch more issues. Plus not overload macos-14 runners queue (if there is one).